pkg/sync/config.go

```diff
@@ -44,6 +46,77 @@ type Config struct {
 	Quiet       bool
 	CheckAll    bool
 	CheckNew    bool
+	Env         map[string]string
+}
+
+func envList() []string {
+	return []string{
+		"ACCESS_KEY",
+		"SECRET_KEY",
+		"SESSION_TOKEN",
+
+		// sync JFS to JFS, but neither has a mount point
+		"myfs",
```

Comment:
we should pass all the variable that contains "JFS"

Reply:
ok